apps/web/src/components/common/select.tsx

```
@@ -3,30 +3,32 @@
 import * as SelectPrimitive from "@radix-ui/react-select"
 import { CheckIcon, ChevronDownIcon, ChevronUpIcon, ChevronsUpDownIcon } from "lucide-react"
 import type { ComponentProps } from "react"
-import { cx } from "~/utils/cva"
+import { cx, popoverAnimationClasses, VariantProps } from "~/utils/cva"
```

Biome (1.9.4)

[error] Some named imports are only used as types.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)